src/components/CardList.tsx

```diff
@@ -1,132 +1,110 @@
-  align-items: center;
-  margin-right: 1rem;
-  flex-wrap: wrap;
-`
-
-const Image = styled(GatsbyImage)`
-  min-width: 20px;
-  margin-right: 1rem;
-  margin-top: 4px;
-`
+import * as url from "../utils/url"
+import Link from "./Link"
 
 export type CardListItem = {
   title?: ReactNode
   description?: ReactNode
   caption?: ReactNode
   link?: string
-  id?: string | number
+  id?: string
 } & ImageProp
 
 export interface IProps {
   content: Array<CardListItem>
-  className?: string
   clickHandler?: (idx: string | number) => void
 }
 
-const CardList: React.FC<IProps> = ({
-  content,
-  className,
-  clickHandler = () => null,
-}) => (
-  <Table className={className}>
+const CardContainer = (props: StackProps) => {
+  return (
+    <HStack
+      spacing={4}
+      p={4}
+      color="text"
+      border="1px solid"
+      borderColor="border"
+      _hover={{
+        borderRadius: "base",
+        boxShadow: "0 0 1px var(--eth-colors-primary)",
+        background: "tableBackgroundHover",
+      }}
+      {...props}
+    />
+  )
+}
+
+const Card = (props: CardListItem & Omit<StackProps, "title" | "id">) => {
+  const { title, description, caption, link, image, alt, ...rest } = props
+
+  const isLink = !!link
+  const isExternal = url.isExternal(link || "")
+
+  return (
+    <CardContainer {...rest}>
+      {image && <Box as={GatsbyImage} image={image} alt={alt} minW="20px" />}
+      <Flex flex="1 1 75%" direction="column">
+        {isLink ? (
+          <LinkOverlay
+            as={Link}
+            href={link}
+            hideArrow
+            color="text"
+            textDecoration="none"
+            _hover={{ textDecoration: "none" }}
+          >
+            {title}
+          </LinkOverlay>
+        ) : (
+          <Box>{title}</Box>
+        )}
+
+        <Box fontSize="sm" mb={0} opacity={0.6}>
+          {description}
+        </Box>
+      </Flex>
+      {caption && (
+        <Flex flex="1 0 25%" align="center" wrap="wrap" mr={4}>
+          <Box fontSize="sm" mb={0} opacity={0.6}>
+            {caption}
+          </Box>
+        </Flex>
+      )}
+      {isExternal && <Box>↗</Box>}
+    </CardContainer>
+  )
+}
+
+const CardList: React.FC<IProps> = ({ content, clickHandler = () => null }) => (
+  <Box bg="background" width="full">
     {content.map((listItem, idx) => {
-      const { title, description, caption, link, image, alt, id } = listItem
+      const { link, id } = listItem
       const isLink = !!link
-      return isLink ? (
-        <ItemLink key={id || idx} to={link}>
-          {image && <Image image={image} alt={alt} />}
-          <LeftContainer>
-            <ItemTitle>{title}</ItemTitle>
 
-            <ItemDesc>{description}</ItemDesc>
-          </LeftContainer>
-          {caption && (
-            <RightContainer>
-              <ItemDesc>{caption}</ItemDesc>
-            </RightContainer>
-          )}
-        </ItemLink>
+      return isLink ? (
+        <LinkBox key={id || idx}>
```

Comment:
More about `LinkOverlay` and `LinkBox` here: https://chakra-ui.com/docs/components/link-overlay/usage

Good for a11y.